frontend/src/util/index.ts

```diff
@@ -37,6 +37,27 @@ export function characterClass(characters: string): string {
     return `[${characters}]`;
 }
 
+/** Checks whether or not a given string is contained in another string */
+export const localeContains = (mainString: string, stringToCheck: string) => {
+    if (stringToCheck === "") {
+        return true;
+    }
+    if (!stringToCheck || !mainString.length) {
+        return false;
+    }
+
+    stringToCheck = "" + stringToCheck;
```

Comment:
This part of the StackOverflow answer is just for converting non string arguments to strings. But since we use Typescript and this function requires `string`, this is not necessary